src/dropdown/Dropdown.js

```diff
@@ -20,34 +20,24 @@ const PropTypes = {
 @props(PropTypes, { strict: false })
 export default class Dropdown extends React.Component {
 
+  componentDidMount() {
+    this.logWarnings();
+  }
+
   getChildren = () => [].concat(this.props.children || []);
 
   renderOption = (option) => this.getChildren()[option.value];
 
   renderValue = (option) => this.getChildren()[option.value];
 
-  getGeneralProps = () => {
-    if (this.props.children && this.props.options) {
-      warn('You\'re passing both children and options. Children will override options!');
-    }
-    return omit(this.props, Object.keys(PropTypes));
-  };
-
-  getChildrenProps = () => {
-    if (this.props.children) {
-      const options = this.getChildren().map((c, index) => {
-        return {
-          value: index,
-          label: index
-        };
-      });
+  getGeneralProps = () => omit(this.props, Object.keys(PropTypes));
 
-      return {
-        options,
-        valueRenderer: this.renderValue,
-        optionRenderer: this.renderOption
-      };
-    }
+  logWarnings = () => {
+    warn(() => {
```

Comment:
ok, i will just have to revert this last commit then. I did it in that way at first, but i wasn't sure